iwyu_test_util.py

```diff
@@ -410,11 +423,24 @@ def _CompareExpectedAndActualSummaries(expected_summaries, actual_summaries):
 
 def _GetLaunchArguments(cc_file):
   """Gets IWYU launch arguments for a source file from its contents."""
+  args = ''
   with open(cc_file) as fh:
-    for line in fh:
+    for (lineno, line) in enumerate(fh, start=1):
       m = _IWYU_TEST_RUN_ARGS_RE.match(line)
       if m:
-        return m.group(1)
+        args = m.group(1)
+
+        comments = itertools.takewhile(_IsCommentLine, fh)
+        
+        while len(args) > 0 and args[-1] == '\\':
+          args = args[:-1] # delete trailing slash
+          try:
+            line = next(comments)
+          except StopIteration:
+            raise SyntaxError('{}:{} syntax error in multiline IWYU_ARGS'.format(cc_file, lineno))
+          args += ' ' + _GetCommentLineContents(line)
```

Comment:
The `_GetCommentLineContents` functions seems a little over-abstracted, I'd probably just go with `line.lstrip('/').lstrip()`